pkgs/by-name/po/powersploit/package.nix

```diff
@@ -0,0 +1,35 @@
+{ lib
+, fetchFromGitHub
+, stdenv
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "powersploit";
+  version = "3.0.0";
+
+  src = fetchFromGitHub {
+    owner = "PowerShellMafia";
+    repo = "PowerSploit";
+    rev = "v${finalAttrs.version}";
+    hash = "sha256-cFj4IwDyiwQDSYLCcloMQ83T8MOtMcKsyMilfrZOyHs=";
+  };
+
+  installPhase = ''
+    runHook preInstall
+
+    mkdir -p $out/share/powersploit
+    cp -a --no-preserve=ownership * $out/share/powersploit
+
+    runHook postInstall
+  '';
+
+  meta = with lib; {
+    changelog = "https://github.com/PowerShellMafia/PowerSploit/releases/tag/v${version}";
+    description = "A PowerShell Post-Exploitation Framework";
+    mainProgram = "powersploit";
```

Comment:
this package doesn't contain any binaries, `mainProgam` is intended to locate main executable binary for a package